modules/carto/src/layers/raster-layer.ts

```diff
@@ -31,7 +31,7 @@ class RasterColumnLayer extends ColumnLayer {
   getShaders() {
     const shaders = super.getShaders();
     const data = this.props.data as unknown as Raster & {length: number};
-    const BLOCK_WIDTH = data.blockWidth || Math.sqrt(data.length);
+    const BLOCK_WIDTH = data.blockSize || Math.sqrt(data.length);
```

Comment:
Could you double check the logic here? We [have a bug](https://github.com/visgl/deck.gl/issues/8587) and I suspect that `Math.sqrt(data.length)` should in fact be `Math.sqrt(data.data.length)`